[2024-01-25T22:01:19.402-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-25T06:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-25T22:01:19.452-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-25T06:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-25T22:01:19.469-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-25T06:00:00+00:00 [queued]>
[2024-01-25T22:01:19.479-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-25T06:00:00+00:00 [queued]>
[2024-01-25T22:01:19.480-0300] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-01-25T22:01:19.507-0300] {taskinstance.py:1382} INFO - Executing <Task(LambdaInvokeFunctionOperator): invoke_lambda_function> on 2024-01-25 06:00:00+00:00
[2024-01-25T22:01:19.509-0300] {standard_task_runner.py:57} INFO - Started process 1752 to run task
[2024-01-25T22:01:19.511-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-03', 'invoke_lambda_function', 'scheduled__2024-01-25T06:00:00+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpk6iy6eu9']
[2024-01-25T22:01:19.511-0300] {standard_task_runner.py:85} INFO - Job 209: Subtask invoke_lambda_function
[2024-01-25T22:01:19.567-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-25T06:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-25T22:01:19.569-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-25T06:00:00+00:00 [running]> on host 14111-NB.
[2024-01-25T22:01:19.676-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='how-desafio-03' AIRFLOW_CTX_TASK_ID='invoke_lambda_function' AIRFLOW_CTX_EXECUTION_DATE='2024-01-25T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-25T06:00:00+00:00'
[2024-01-25T22:01:19.677-0300] {lambda_function.py:211} INFO - Invoking AWS Lambda function: how-desafio-03 with payload: {"weather_daily": {"date": "1706230800"}}
[2024-01-25T22:01:19.686-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-25T22:01:19.687-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-25T22:01:20.609-0300] {lambda_function.py:220} INFO - Lambda response metadata: {'RequestId': 'c800c45e-41a2-46b9-8dc9-097665da0007', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 01:01:41 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'c800c45e-41a2-46b9-8dc9-097665da0007', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b30475-2b5144a35884bc9c18437fa5;sampled=0'}, 'RetryAttempts': 0}
[2024-01-25T22:01:20.610-0300] {lambda_function.py:230} INFO - Lambda function invocation succeeded: {'RequestId': 'c800c45e-41a2-46b9-8dc9-097665da0007', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 01:01:41 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'c800c45e-41a2-46b9-8dc9-097665da0007', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b30475-2b5144a35884bc9c18437fa5;sampled=0'}, 'RetryAttempts': 0}
[2024-01-25T22:01:20.660-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-03, task_id=invoke_lambda_function, execution_date=20240125T060000, start_date=20240126T010119, end_date=20240126T010120
[2024-01-25T22:01:20.690-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-25T22:01:20.704-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
